configure.ac

```diff
@@ -71,6 +71,38 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
 
 AC_LANG_POP(C)
 
+AC_ARG_ENABLE([authy],
+	[AC_HELP_STRING([--enable-authy],
+			[enable Authy push authentcation [default=no]])],
+	[
+		xenable_authy=$enableval
+		AC_DEFINE(ENABLE_AUTHY, [1], [Enable Authy extension])
+	],
+	[xenable_authy=no])
+AM_CONDITIONAL([ENABLE_AUTHY], [false])
+
+AC_CHECK_LIB([curl], [curl_global_init],
```

Comment:
I'm not an autoconf expert, but why not just `AC_CHECK_LIB([curl], [curl_global_init])` ?

Reply:
Agreed, I'm going to rewrite configure.ac